tb/cocotb/gpu/gpu_ref_model.py

```
@@ -59,64 +123,172 @@ def run(self, words: dict, max_steps: int = 10000) -> dict:
 
+            # Only active lanes contribute to reads/writes.
             a = self.regs[rs1]
             b = self.regs[rs2]
 
             if opcode == VRET:
-                break
+                if div_stack:
+                    # Pop: resume false-path (not-taken lanes)
+                    return_pc, return_mask_bits = div_stack.pop()
+                    # Restore active mask from the bitmask stored on stack
+                    active_mask = [l for l in range(self.n)
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)